providers/implementations/digests/sha3_prov.c

```diff
@@ -177,11 +177,18 @@ static int s390x_shake_final(unsigned char *md, void *vctx)
     return 1;
 }
 
+size_t SHA3_absorb(uint64_t A[5][5], const unsigned char *inp, size_t len,
+                   size_t r);
+
+void SHA3_squeeze(uint64_t A[5][5], unsigned char *out, size_t len, size_t r);
+
 static int s390x_keccakc_final(unsigned char *md, void *vctx, int padding)
 {
     KECCAK1600_CTX *ctx = vctx;
     size_t bsz = ctx->block_size;
     size_t num = ctx->bufsz;
+    size_t needed = ctx->md_size;
+    unsigned char empty[200] = {0};
```

Comment:
What about using `KECCAK1600_WIDTH / 8` instead of 200?

Also the declaration should be `const static unsigned char`.

Reply:
Thanks for the suggestion.  Changed both as suggested.